website/templates/project/registration_editor_templates.mako

```diff
@@ -4,9 +4,18 @@
 </script>
 
 <script type="text/html" id="text">
-  <input data-bind="valueUpdate: 'keyup',
-                    value: value"
-         type="text" class="form-control" />
+    <span data-bind="if: required">
+        <input data-bind="valueUpdate: 'keyup',
+                          value: value,
+                          tooltip: {title: 'This field is required for submission. If this field is not applicable to your study, you may state so.', placement: 'right'}"
```

Comment:
I'm not sure I understand what part of the form this corresponds to. I see the tooltip defined below when I hover or click on a text box, but I'm not sure where this one is happening.